GameFiles/_integration.gsc

```diff
@@ -544,45 +604,18 @@ OnExecuteCommand( event )
     data = ParseDataString( event.data );
     response = "";
 
-    switch ( event.subtype ) 
-    {
-        case "GiveWeapon":
-            response = event.target GiveWeaponImpl( data );
-            break;
-        case "TakeWeapons":
-            response = event.target TakeWeaponsImpl();
-            break;
-        case "SwitchTeams":
-            response = event.target TeamSwitchImpl();
-            break;
-        case "Hide":
-            response = self HideImpl();
-            break;
-        case "Unhide":
-            response = self UnhideImpl();
-            break;
-        case "Alert":
-            response = event.target AlertImpl( data );
-            break;
-        case "Goto":
-            if ( IsDefined( event.target ) )
-            {
-                response = self GotoPlayerImpl( event.target );
-            }
-            else
-            {
-                response = self GotoImpl( data );
-            }
-            break;
-        case "Kill":
-            response = event.target KillImpl();
-            break;
-        case "NightMode":
-            NightModeImpl();
-            break;
-        case "SetSpectator":
-            response = event.target SetSpectatorImpl();
-            break;
+    cmd = level.clientCommandCallbacks[event.subtype];
+    runAsTarget = level.clientCommandRusAsTarget[event.subtype];
+    context = event.origin;
```

Comment:
might be wordy but consider renaming this to `executionContextEntity` or something, to be a little more explicit what the "context" is